vscode-extension/package.json

```diff
@@ -64,9 +67,16 @@
     "webpack-cli": "^4.4.0"
   },
   "dependencies": {
-    "@types/squirrelly": "^7.1.1",
-    "fs-extra": "^9.1.0",
+    "@oliveai/loop-templates": "1.0.4",
```

Comment:
I noticed `@oliveai/loop-templates` in the `devDependencies` too. I think it makes sense to be in `dependencies`, but you probably want to use `"file:../loop-templates"` for the value (and remove it from `devDependencies`

Reply:
@racary-olive You're right that it gets published, though I'm not sure why. We want the two tools (VSCode extension and `npx` command) in this repo to reference the **current** state of the `loop-templates` dir. Otherwise, it becomes complicated/difficult to test changes to `loop-templates` in these tools. If each of these 3 things were in separate repositories and had different release schedules, it would make sense to (and you'd have to) point to a specific version. I think they used to be in different repos, which is why it was set up to be published to NPM. But since they're all in the same repo now, it's intentional that both tools reference the templates as a relative path dir, rather than a specific version. I'm happy to explain more if that doesn't clear it up

Reply:
thanks for the explanation! that makes total sense. ill update

Reply:
@iangallOlive it looks like publishing the loop-templates is a part of the github action that runs before building and publishing the vscode extension.

Im fine with using the file:../loop-templates but i think we will need to update the github actions to account for building the loop-templates

Reply:
I just confirmed with someone that there's no need to publish `loop-templates` to NPM. It was necessary for something in the past, but no longer true. We can either remove it being published to NPM from the build pipeline, or just let it continue the way it is (there's no harm in publishing it to NPM). But either way, the `npx` and VSCode projects need to reference it with `file:../loop-templates`

Reply:
I think removing that step from the pipeline make sense then and update the pipeline to just build the loop-templates folder

Reply:
This [line](https://github.com/open-olive/vscode-loop-development-kit/blob/main/vscode-extension/src/loopCreator.ts#L7) references a file within `loop-templates/dist`. I think that only exists if loop-templates gets built

Reply:
Yeah i agree, i actually pushed changes to just pull the file. And yes everything else is handled & correct